sickbeard/providers/__init__.py

```diff
@@ -107,4 +106,5 @@ def getProviderClass(provider_id):
         for x in sickbeard.providerList + sickbeard.newznabProviderList + sickbeard.torrentRssProviderList
         if x
     }
-    return providers[provider_id]
+    if providers:
+        return providers.values()[0]
```

Comment:
@medariox this change is making all providers in hstory appear with NZB image.

Also why getProviderClass used to return a list and now returns a dict?!
this change will need to be one in all over Medusa code!